spring-integration-zip/src/test/java/org/springframework/integration/zip/splitter/UnZipResultSplitterTest-context.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:int="http://www.springframework.org/schema/integration"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+
+
+
+		http://www.springframework.org/schema/integration http://www.springframework.org/schema/integration/spring-integration.xsd">
```

Comment:
Strange formatting, but I'm sure can be fix with the reformat tool in the IDE.
And again tab not whitespaces.
You can find some common Spring Projects IDE configuration options here:
https://github.com/spring-projects/spring-framework/tree/master/src/idea
https://github.com/spring-projects/spring-framework/tree/master/src/eclipse